tests/backup/backup_multiple_provisioner_test.go

```diff
@@ -222,3 +223,411 @@ var _ = Describe("{MultipleProvisionerCsiSnapshotDeleteBackupAndRestore}", Label
+			err = DeleteScheduleWithUIDAndWait(kdmpScheduleName, scheduleUid, SourceClusterName, srcClusterUid, BackupOrgID, ctx)
+		})
+		Step("Taking manual backup of application from source cluster", func() {
+			log.InfoD("taking manual backup of applications")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			backupNames = make([]string, 0)
+			if len(provisionerDefaultSnapshotClassMap) > 0 {
+				for i, appCtx := range scheduledAppContextsForDefaultVscBackup {
+					scheduledNamespace := appCtx.ScheduleOptions.Namespace
+					backupName := fmt.Sprintf("%s-%s-%v", "autogenerated-backup", scheduledNamespace, time.Now().Unix())
+					provisionerVolumeSnapshotClassSubMap := make(map[string]string)
+					if value, ok := provisionerDefaultSnapshotClassMap[appCtx.ScheduleOptions.StorageProvisioner]; ok {
+						provisionerVolumeSnapshotClassSubMap[appCtx.ScheduleOptions.StorageProvisioner] = value
+					}
+					log.InfoD("creating backup [%s] in source cluster [%s] (%s), organization [%s], of namespace [%s], in backup location [%s]", backupName, SourceClusterName, srcClusterUid, BackupOrgID, scheduledNamespace, backupLocationName)
+					err = CreateBackupWithValidationWithVscMapping(ctx, backupName, SourceClusterName, backupLocationName, backupLocationUID, scheduledAppContextsForDefaultVscBackup[i:i+1], labelSelectors, BackupOrgID, srcClusterUid, preRuleNameDefaultVsc, preRuleUidDefaultVsc, postRuleNameDefaultVsc, postRuleUidDefaultVsc, provisionerVolumeSnapshotClassSubMap, false)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of backup [%s]", backupName))
+					backupNames = append(backupNames, backupName)
+				}
+			} else {
+				log.InfoD("Skipping this step as provisioner with default volumeSnapshotClass is not found")
+			}
+		})
+		Step("Restoring the backup taken on singe namespace with multiple application deployed with different provisioner", func() {
+			if multipleNsSchBackupName != "" {
+				log.InfoD("Restoring the backup taken on singe namespace with multiple application deployed with different provisioner")
+				ctx, err := backup.GetAdminCtxFromSecret()
+				log.FailOnError(err, "Fetching px-central-admin ctx")
+				namespaceMappingMultiApp := make(map[string]string)
+				for _, appCtx := range scheduledAppContextsForMultipleAppSinleNs {
+					namespaceMappingMultiApp[appCtx.ScheduleOptions.Namespace] = appCtx.ScheduleOptions.Namespace + "-mul-app-snigle-ns"
+				}
+				restoreName := fmt.Sprintf("%s-%s-%s", "test-restore", "multi-app-single-ns", RandomString(randomStringLength))
+				log.InfoD("Restoring namespaces from the [%s] backup", multipleNsSchBackupName)
+				err = CreateRestoreWithValidation(ctx, restoreName, multipleNsSchBackupName, namespaceMappingMultiApp, make(map[string]string), DestinationClusterName, BackupOrgID, scheduledAppContextsForMultipleAppSinleNs)
```

Comment:
Restore verification is missing, Same in below restores as well.

Reply:
done